patches/0013-remove-long-path-support-hack.patch

```diff
@@ -0,0 +1,56 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: qmuntal <qmuntaldiaz@microsoft.com>
+Date: Thu, 21 Mar 2024 11:36:42 +0100
+Subject: [PATCH] remove long path support hack
+
+---
```

Comment:
It would be good to mention the reason and consequences here, IMO.

Reply:
Good point. Done!